src/utils/queries/queries.js

```diff
@@ -3,7 +3,7 @@ import * as queryTypes from './queryTypes';
 
 const queries = {
   getUsers: () => gqlQuery(queryTypes.GET_USERS),
-  newUser: (user) => gqlMutate(queryTypes.NEW_USER, user)
-}
+  newUser: user => gqlMutate(queryTypes.NEW_USER, user)
```

Comment:
Single arguments that are passed into arrow functions are required to have no parens!